storm/String.cpp

```diff
@@ -500,7 +500,7 @@ size_t STORMCDECL SStrPrintf(char* dest, size_t maxchars, const char* format, ..
     return ISStrVPrintf(dest, maxchars, format, va);
 }
 
-size_t STORMAPI SStrVPrintf(char* dest, size_t maxchars, const char* format, va_list arglist) {
+size_t STORMCDECL SStrVPrintf(char* dest, size_t maxchars, const char* format, va_list arglist) {
```

Comment:
Even though it's not variadic, it's `__cdecl` in storm.dll.

Reply:
Huh. I'll have to mull over this at some point. But I'm game with it for now.